packages/grafana-data/src/transformations/fieldReducer.test.ts

```diff
@@ -55,14 +55,13 @@ describe('Stats Calculators', () => {
   it('should calculate basic stats', () => {
     const stats = reduceField({
       field: basicTable.fields[0],
-      reducers: [ReducerID.first, ReducerID.last, ReducerID.mean, ReducerID.count, ReducerID.diffperc],
+      reducers: [ReducerID.first, ReducerID.last, ReducerID.mean, ReducerID.count],
     });
 
     expect(stats.first).toEqual(10);
     expect(stats.last).toEqual(20);
     expect(stats.mean).toEqual(15);
     expect(stats.count).toEqual(2);
-    expect(stats.diffperc).toEqual(100);
```

Comment:
we can probably keep this test and just update this result to be `1` instead of `100` - this would help us make sure we acknowledge the handling of this value changing over time so probably worth to keep the test 🤷